mne/viz/topomap.py

```diff
@@ -1134,19 +1135,34 @@ def plot_evoked_topomap(evoked, times=None, ch_type=None, layout=None,
 
     if isinstance(axes, plt.Axes):
         axes = [axes]
-    if times is None:
+
+    if times == "peaks":
+        from scipy.signal import argrelmax
+        gfp = evoked.data.std(axis=0)
+        order = int(evoked.info["sfreq"] / 5.)
+        peaks = argrelmax(gfp, order=order)
+        if len(peaks) > 10:
+            peaks = peaks[np.sort((-peaks).argsort()[:10])]
```

Comment:
The length of `peaks` I think is determined entirely by `len(times)` and `order`, so you shouldn't need a while loop to figure it out. But I also don't like this approach -- imagine you have 10 sec experiment where everything happens in the first 1 sec. Pushing `order` higher forces you take take peaks from each 1 sec interval when really you might be more interested in the multiple peaks in the first second. So to me it would be better to choose `order` based on the `lowpass` and take whatever peaks you get (distributed through time however they are) using the code I posted above.